app.py

```diff
@@ -26,13 +27,31 @@ def post_random_game_from_users_collection() -> Union[str, Response]:
     bgg_companion_api = BggCompanionApi(request_client=RequestsRetryClient())
     try:
         filtered_board_games = bgg_companion_api.get_users_filtered_board_games(user)
-        return jsonify(dataclasses.asdict(random.choice(filtered_board_games)))
+        resp = jsonify(dataclasses.asdict(random.choice(filtered_board_games)))
+        resp.set_cookie(key="username", value=user)
```

Comment:
nit: can we move the cookie names to constants, like `USERNAME_COOKIE_NAME` and `VISIT_COUNT_COOKIE_NAME`? prefer to store magic stings in variables instead of repeating them all over the place

Reply:
Hmmm, what would be renamed?  in this case?  `resp` is the Response object not the cookie itself so I don't think I'd change that variable name right?  And I'm just using the set_cookie method within the Response class.

Reply:
Hmmm @nickdibari , what would be renamed? in this case? resp is the Response object not the cookie itself so I don't think I'd change that variable name right? And I'm just using the set_cookie method within the Response class.

Reply:
OHHHHH I understand.  Just pushed the change!